scripts/gsc-indexing-audit.py

```
@@ -0,0 +1,290 @@
+                    f"      google: {google_canonical}"
+                )
+            time.sleep(1)  # be gentle with quota
+        except HttpError as e:
+            print(f"[{bucket}] {url}  → ERROR: {e}")
+
+
+def main():
+    gsc = authed()
+
+    sitemap_stats(gsc)
+    cur, prev, cur_agg, prev_agg = pattern_aggregation(gsc)
```

Ruff (0.15.12)

[warning] Unpacked variable `cur_agg` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)